File: platforms/documentation/docs/src/samples/templates/java-utilities-library/src/main/java/org/gradle/sample/utilities/StringUtils.java
package org.gradle.sample.utilities;

import org.gradle.sample.list.ArrayList;

public class StringUtils {
    public static String join(ArrayList source) {
        return JoinUtils.join(source);
    }

    public static ArrayList split(String source) {
        return SplitUtils.split(source);
    }
}

File: subprojects/docs/src/samples/templates/kotlin-utilities-library/src/main/kotlin/org/gradle/sample/utilities/StringUtils.kt
package org.gradle.sample.utilities

import org.gradle.sample.list.LinkedList

class StringUtils {
    companion object {
        fun join(source: LinkedList): String {
            return JoinUtils.join(source)
        }

        fun split(source: String): LinkedList {
            return SplitUtils.split(source)
        }
    }
}
